### PR TITLE
Add more webistes [Update data.json]

### PR DESCRIPTION
**Pull request by DjamelEd**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
Thanks! That looks better! Will get to looking at it soon!

**Comment by deamme** [comment]
You can accept BTC as a merchant right? But shopify itself doesn't accept BTC payments in any form. What could be nice to add is the merchants themselves that accepts BTC.

For example those listed here: https://www.shopify.com/bitcoin

File: data/data.json [hunk 1]
```diff
@@ -10,5 +10,35 @@
...
+  },
+  {
+    "title": "Belovedshirts",
+    "link": "https://www.belovedshirts.com/",
+    "category": "E-commerce",
+    "crypto": ["BTC"]
+  },
+  {
+    "title": "Enclavemfg",
+    "link": "https://www.enclavemfg.com//",
```
Comment (deamme) [inline data/data.json]: Typo here `//`

**Comment by deamme** [comment]
I modified the `data.json` a bit so now we have conflicting changes. Can you please resolve them? Sorry about that.

**Comment by deamme** [comment]
@DjamelEd `category` is renamed to `categories` which takes an array now. Can you resolve this too?

File: data/data.json [hunk 2]
```diff
@@ -14,31 +14,31 @@
...
     "link": "https://www.newegg.com",
-    "category": "Electronics",
+    "categories": ["Electronics"],
     "crypto": ["BTC"]
   },
 {
     "title": "Travala",
     "link": "https://www.travala.com",
-    "category": "",
+    "categories": [],
```
Comment (FrederikBolding) [inline data/data.json]: Please add a category here! 😄

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

**Pull request merged**